### PR TITLE
release v0.8.0

### PR DESCRIPTION
**Pull request by jiangpengcheng**
*(If this PR fixes a github issue, please add `Fixes #<xyz>`.)*

Fixes #<xyz>

*(or if this PR is one task of a github issue, please add `Master Issue: #<xyz>` to link to the master issue.)*

Master Issue: #<xyz>

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

- [x] This change is a trivial rework / code cleanup without any test coverage.

### Documentation

Check the box below.

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [x] `no-need-doc` 
  
  release pr
  
- [ ] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
 # Function Mesh v0.7.0 Release Notes
 
+## v0.8.0 What's New
+
+* [Controller] upgrade the k8s dependency libraries to v1.22.\*. ([#494](https://github.com/streamnative/function-mesh/pull/494))
```
Comment (tpiperatgod) [inline CHANGELOG.md:5]: I suggest that the first letter be upper case

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+* [Controller] Resolve the namespace conflict of protocol buffers package ([#517](https://github.com/streamnative/function-mesh/pull/517))
+* [Controller] Add a flag to enable/disable downloader ([#515](https://github.com/streamnative/function-mesh/pull/515))
+* [Controller] Add java opts ([#484](https://github.com/streamnative/function-mesh/pull/484))
+* [Controller] Fix legacy authSecret and tlsSecret error ([#523](https://github.com/streamnative/function-mesh/pull/523))
+* [CRD] Add finalizers to RBAC ([#485](https://github.com/streamnative/function-mesh/pull/485))
+* [CRD] Migrate ClusterRole from v1beta1 to v1 ([#492](https://github.com/streamnative/function-mesh/pull/492))
+* [CRD] Upgrade OLM operator capability level to "Full Lifecycle" ([#518](https://github.com/streamnative/function-mesh/pull/518))
+* [CRD] Reserve unknown fields for batchSourceConfig ([#513](https://github.com/streamnative/function-mesh/pull/513))
+* [Doc] Add readme docs to helm chart ([#497](https://github.com/streamnative/function-mesh/pull/497))
+* [Makefile] adopt openshift olm annotation ([#488](https://github.com/streamnative/function-mesh/pull/488))
```
Comment (tpiperatgod) [inline CHANGELOG.md:20]: ditto

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,29 @@
...
+* [Test] Fix release bug ([#520](https://github.com/streamnative/function-mesh/pull/520))
+* [Test] Change e2e to pull_request_target trigger type ([#519](https://github.com/streamnative/function-mesh/pull/519))
+
+For the full changes in this release, see the [Pull Requests](https://github.com/streamnative/function-mesh/pulls?q=is%3Apr+label%3Am%2F2022-11+is%3Aclosed
```
Comment (tpiperatgod) [inline CHANGELOG.md:24]: this link is broken

File: charts/function-mesh-operator/README.md [hunk 4]
```diff
@@ -20,34 +20,35 @@ The Function Mesh operator Helm chart for Kubernetes
...
-| controllerManager.resources.requests.memory | string | `"50Mi"` |
-| controllerManager.selector | list | `[]` |
-| controllerManager.serviceAccount | string | `"function-mesh-controller-manager"` |
-| controllerManager.tolerations | list | `[]` |
-| imagePullPolicy | string | `"IfNotPresent"` |
-| imagePullSecrets | list | `[]` |
-| installation.namespace | string | `"function-mesh-system"` |
-| operatorImage | string | `"streamnative/function-mesh:v0.7.0"` |
-| rbac.create | bool | `true` |
+| Key | Type | Default | Description |
```
Comment (tpiperatgod) [inline charts/function-mesh-operator/README.md:27]: It is recommended to delete the "Description" column.

File: charts/function-mesh-operator/charts/admission-webhook/README.md [hunk 5]
```diff
@@ -1,28 +1,28 @@
...
 * <https://github.com/streamnative/function-mesh>
 
 ## Values
 
-| Key | Type | Default |
-|-----|------|---------|
-| admissionWebhook.certificate.provider | string | `"cert-manager"` |
-| admissionWebhook.enabled | bool | `true` |
-| admissionWebhook.failurePolicy | string | `"Fail"` |
+| Key | Type | Default | Description |
```
Comment (tpiperatgod) [inline charts/function-mesh-operator/charts/admission-webhook/README.md:21]: It is recommended to delete the "Description" column.

File: charts/function-mesh-operator/README.md [hunk 6]
```diff
@@ -1,16 +1,16 @@
...
+function mesh operator Helm chart for Kubernetes
 
 **Homepage:** <https://github.com/streamnative/function-mesh>
 
 ## Maintainers
 
-| Name | Email                                | URL |
-| ---- |--------------------------------------| --- |
-| Function Mesh | mailto:function-mesh@streamnative.io | https://github.com/streamnative/function-mesh |
+| Name | Email | Url |
```
Comment (tpiperatgod) [inline charts/function-mesh-operator/README.md:11]: I think we can follow @Huanli-Meng 's suggestions, pls refer to this: https://github.com/streamnative/function-mesh/pull/497#pullrequestreview-1161363574

File: charts/function-mesh-operator/README.md (hunk 4 above)
Comment (jiangpengcheng) [reply charts/function-mesh-operator/README.md:27]: it's generated by helm-docs

**Review by tpiperatgod — APPROVED** [review]
lgtm

**Pull request merged**